tailwind.config.ts

```diff
@@ -19,8 +20,33 @@ const config: Config = {
   darkMode: "class",
   plugins: [
     require("@tailwindcss/typography"),
+
+    // when extendning colors, "" wipes out everything,
```

Comment:
tidy this up

Reply:
// when extendning colors, "" wipes out everything,
    // so comment out unused lines